### DISCUSSION TITLE
Net usage?

### DISCUSSION THREAD

**Discussion by ViktorJp** [jasonacox/Powerwall-Dashboard#138] [Ideas]
Unless I'm missing something... one of the most useful data points I like to refer to on the Tesla app is the "net usage" field, which is basically the difference between "from Grid" and "to Grid"... giving me a good picture of if I'm in the positive or negative for the day.  Flipping from day to day, gives me a good indication if the month looks good for a negative electric bill or not.  ;)  It seems that most of the data available on the Powerwall Dashboard are stats that cover the last 24hrs, to the minute... so they constantly change throughout the day... 

As a feature change or addition, it would be nice to see a few static fields showing the "net usage" for the day (from the 00:00 up to the 24:00 point of the day), and maybe the current "from grid"/"to grid" stats for the same period/day?  I'm not seeing the usefulness of the previous period/days stats interfering with the stats of today?  Know what I mean?  Perhaps adding another counter to the "power meters" area that covers this?

If I can help explain things further, please let me know.... ;)

**Comment by jasonacox** [comment]
Hi @ViktorJp  - correct me if I'm wrong, but you are just wanting the "Today" selection in the the duration drop-down (top right corner).  You can set "Today" as your default view by also clicking the Save icon:

<img width="1061" alt="image" src="https://user-images.githubusercontent.com/836718/203892090-fa3a4566-507c-4343-be64-70c9b0ec19b1.png">

You can also see day over day grid usage (from/to) in the monthly view which also plots current day:

<img width="517" alt="image" src="https://user-images.githubusercontent.com/836718/203893203-a84eed1e-b6c8-45f3-a02c-180e97e9dee2.png">

**Reply by mcbirse** [reply]
This is how I prefer mine actually. I have my default dashboard set to "Today" and saved like that.

I also made some other personal preference tweaks as well... such as showing the Temperature units (C) and re-scaling my temperature so it is shown more prominently on the graph (i.e. rescaled so 50% would be 25 degrees C - it is very easy at a glance to understand the temperature since is is simply half the % scale value).

![image](https://user-images.githubusercontent.com/108725631/203893632-ee85e27b-4357-4ea4-af04-497614e97bb5.png)

**Comment by ViktorJp** [comment]
Thanks so much for correcting me on that, @jasonacox and @mcbirse!  I never even thought to click on that in order to select "today". Lol.  Ahhhh so much better!!  Much appreciated!
